src/wp-error-handling.php

```diff
@@ -0,0 +1,243 @@
+<?php
+/**
+ * Set up basic error handling for WordPress.
+ *
+ * Initially we simply want to avoid error messages accidentally containing
+ * HTML in certain configurations. In the future we may improve the display
+ * beyond what PHP does by default.
```

Comment:
I don't think we need "In the future we may improve the display beyond what PHP does by default."

Rather than speculate about what might be in this class at some point later, we should just describe what is in the class currently.